[13.0] [MIG] sale_stock_sourcing_address

nikul-serpentcs opened the pull request.

HviorForgeFlow committed:
sale_stock_sourcing_address: Migration to 12.0

oca-travis committed:
Update sale_stock_sourcing_address.pot

OCA-git-bot committed:
README.rst

nikul-serpentcs commented:
@JordiBForgeFlow @HviorForgeFlow @AaronHForgeFlow Can you please take a look?

HviorForgeFlow reviewed (approved):
LGTM :+1:

rousseldenis reviewed (approved):
Code review

AaronHForgeFlow commented on sale_stock_sourcing_address/readme/CONTRIBUTORS.rst:
```diff
@@ -0,0 +1,5 @@
+* Eficent
```
ForgeFlow

AaronHForgeFlow commented on sale_stock_sourcing_address/models/account_invoice_line.py:
```diff
@@ -0,0 +1,20 @@
+# Copyright 2019 Eficent Business and IT Consulting Services, S.L.
```
ForgeFlow S.L.

AaronHForgeFlow reviewed (approved):
Tests ok

nikul-serpentcs commented:
@AaronHForgeFlow @JordiBForgeFlow Ready to merge

JordiMForgeFlow reviewed (approved):
Reviewed

dreispt commented:
/ocabot merge nobump

AaronHForgeFlow commented:
Does this need retry or a fix?

AaronHForgeFlow commented:
@dreispt could you retry?

newtratip commented on sale_stock_sourcing_address/__manifest__.py:
```diff
@@ -0,0 +1,18 @@
+{
+    "name": "Sale Stock Sourcing Address",
+    "version": "13.0.1.0.0",
+    "author": "ForgeFlow S.L.," "Odoo Community Association (OCA)",
```
```suggestion
    "author": "ForgeFlow S.L.,Odoo Community Association (OCA)",
```

rousseldenis commented:
/ocabot merge patch

rousseldenis commented:
/ocabot merge patch

rousseldenis commented:
@nikul-serpentcs Could you rebase ?

AaronHForgeFlow commented:
it seems travis is not happy

danimv5 committed:
Added translation using Weblate (Spanish)

danimv5 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (5 of 5 strings)

Translation: sale-workflow-12.0/sale-workflow-12.0-sale_stock_sourcing_address
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-12-0/sale-workflow-12-0-sale_stock_sourcing_address/es/

danimv5 committed:
Added translation using Weblate (Catalan)

danimv5 committed:
Translated using Weblate (Catalan)

Currently translated at 100.0% (5 of 5 strings)

Translation: sale-workflow-12.0/sale-workflow-12.0-sale_stock_sourcing_address
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-12-0/sale-workflow-12-0-sale_stock_sourcing_address/ca/

rafaelbn commented:
/ocabot migration sale_stock_sourcing_address

rousseldenis commented:
/ocabot rebase

rousseldenis commented:
@nikul-serpentcs @JayVora-SerpentCS

nikul-serpentcs committed:
[MIG] sale_stock_sourcing_address v13

nikul-serpentcs commented:
@rousseldenis now the code is updated, Can you please take a look?

Test case failed based on  **sale_sourced_by_line** module.

MiquelRForgeFlow commented:
@nikul-serpentcs please, edit https://github.com/OCA/sale-workflow/blob/13.0/.github/workflows/test.yml#L37 and add case specific for sale_sourced_by_line, and exclude it in the generic case.

MiquelRForgeFlow commented on .github/workflows/test.yml:
```diff
@@ -49,6 +49,13 @@ jobs:
           - container: ghcr.io/oca/oca-ci/py3.6-ocb13.0:latest
             exclude: "sale_order_line_sequence"
             name: test with OCB
+          - container: ghcr.io/oca/oca-ci/py3.6-odoo13.0:latest
+            exclude: "sale_sourced_by_line"
+            makepot: "true"
+            name: test with Odoo
+          - container: ghcr.io/oca/oca-ci/py3.6-ocb13.0:latest
+            exclude: "sale_sourced_by_line"
+            name: test with OCB
```
```suggestion
            exclude: "sale_order_line_sequence,sale_sourced_by_line"
            name: test with OCB
          - container: ghcr.io/oca/oca-ci/py3.6-odoo13.0:latest
            include: "sale_sourced_by_line"
            makepot: "true"
            name: test with Odoo
          - container: ghcr.io/oca/oca-ci/py3.6-ocb13.0:latest
            include: "sale_sourced_by_line"
            name: test with OCB
```

MiquelRForgeFlow replied:
sorry, add sale_sourced_by_line also in line 46

nikul-serpentcs replied:
Done, @MiquelRForgeFlow Brother :smile:

nikul-serpentcs commented:
Thanks, @MiquelRForgeFlow, Now your turn

MiquelRForgeFlow commented:
all green 🍏 🥳 ready to merge!

MiquelRForgeFlow commented:
@rousseldenis

rousseldenis commented:
> @nikul-serpentcs please, edit https://github.com/OCA/sale-workflow/blob/13.0/.github/workflows/test.yml#L37 and add case specific for sale_sourced_by_line, and exclude it in the generic case.

@nikul-serpentcs @MiquelRForgeFlow FYI, `copier update` command will ask you to add rebel module and will do the job

rousseldenis commented:
/ocabot merge patch
